frontend/pages/courses_id/index.css

```diff
@@ -53,7 +53,7 @@
     padding-bottom: 200px;
     &.-newest-first section.problems{
```

Comment:
This entire section can probably be left out, but to be honest I did not even check the full css nor try it, just noticed it now.